www/src/layouts/index.js

```diff
@@ -44,6 +44,12 @@ class Template extends Component {
       ? {location}
       : undefined;
 
+    // TODO - is there a better way to check if we need we have a sidebar?
```

Comment:
@bvaughn @flarnie Any ideas how to achieve this?

Reply:
Yes. @flarnie or I can help with this though.